chain/cosmos/ics.go

```diff
@@ -365,7 +365,21 @@ func (c *CosmosChain) transformCCVState(ctx context.Context, ccvState []byte, co
 		(semver.Compare(providerVersion, icsVer330) < 0 && semver.Compare(consumerVersion, icsVer330) < 0) {
 		return ccvState, nil
 	}
+
 	var imageVersion, toVersion string
+
+	if icsCfg.ProviderVerOverride != "" {
+		imageVersion = icsCfg.ProviderVerOverride
+	}
+
+	if icsCfg.ConsumerVerOverride != "" {
+		toVersion = icsCfg.ConsumerVerOverride
+	}
+
+	if imageVersion == toVersion && imageVersion != "" {
+		return ccvState, nil
+	}
```

Comment:
allowing override is useful in some cases for special ICS forks (i.e. Ethos) or partial set security